### ISSUE TITLE
bug: something is up with week 7

### ISSUE THREAD

**Issue by hkirat** [code100x/cms#1626]
<img width="950" alt="Screenshot 2024-12-07 at 9 42 40 PM" src="https://github.com/user-attachments/assets/a2ce7dd1-43b1-41be-8cbe-62401a826d54">

**Comment by saurabhraghuvanshii** [comment]
@hkirat  it is because of new pr merged #1486

**Comment by IkramBagban** [comment]
Hey @hkirat @saurabhraghuvanshii, could you please explain what this issue is about? I am not able to understand it. Is it about the title not showing? I think someone has already raised a PR for this.

**Comment by saurabhraghuvanshii** [comment]
@IkramBagban ya I see it is resolve in #1622 . check it sir @hkirat

**Comment by IkramBagban** [comment]
> @IkramBagban ya I see it is resolve in #1622 . check it sir @hkirat

But he mentioned that something is up with week 7.

**Comment by srv-18** [comment]
@IkramBagban inside contents are not displayed in the sidebar of the show content.
![Screenshot 2024-12-11 181527](https://github.com/user-attachments/assets/5ad973bd-2cf2-4346-be97-ee5d2cc47414)

**Comment by IkramBagban** [comment]
> @IkramBagban inside contents are not displayed in the sidebar of the show content. ![Screenshot 2024-12-11 181527](https://private-user-images.githubusercontent.com/144537617/394750558-5ad973bd-2cf2-4346-be97-ee5d2cc47414.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5Mjc2NzMsIm5iZiI6MTczMzkyNzM3MywicGF0aCI6Ii8xNDQ1Mzc2MTcvMzk0NzUwNTU4LTVhZDk3M2JkLTJjZjItNDM0Ni1iZTk3LWVlNWQyY2M0NzQxNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMVQxNDI5MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yOTI0N2JlM2ZkMDQ4MzNjMDljYmM0ZDEzMmJhYTU2NzMzMDIzZDJkMzcxNGFhZjUzZTY2Zjc1MTc3ZTBiMzVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.C4PvPNQmnW2roqSIYeeTmwFtJeXHuyFARoYS6b0RH9Y)

oh. this issue is already raised by someone

**Comment by avparadox** [comment]
hey @devsargam i believe even this belongs to branch #1632.

**Comment by omkarsp45** [comment]
@devsargam 
has this bug been fixed? it seems to be working properly on the development side.

![{45D58412-4BD5-46A2-8BE6-FA6A254272D4}](https://github.com/user-attachments/assets/555f2b6d-7c8b-42ab-8400-5de580ed424b)

**Comment by devsargam** [comment]
Yeah, it has been fixed. but there's issue in the production code somewhere. Let me check and fix it.
